ArmPkg/Universal/Smbios/SmbiosMiscDxe/Type03/MiscChassisManufacturerFunction.c

```diff
@@ -143,7 +143,7 @@ SMBIOS_MISC_TABLE_FUNCTION (MiscChassisManufacturer) {
 
   // ContainedElements
   ASSERT (ContainedElementCount < 2);
-  (VOID)CopyMem (SmbiosRecord + 1, &ContainedElements, ExtendLength);
+  (VOID)CopyMem (&ContainedElements, SmbiosRecord + 1, ExtendLength);

```

Comment:
This doesn't make sense: `SmbiosRecord + 1` points to a zeroed region in the pool allocation.

Shouldn't this be copying the additional ContainedElements from InputData to `SmBiosRecord + 1` if the count > 1 ? (`SMBIOS_TABLE_TYPE3` contains a single element but it is intended to be a flexible array)

Reply:
I actually also have no idea why we would have `ASSERT (InputData->ContainedElementCount < 2);` (modifed version, but same logic as before, I believe).

@bexcran I believe your commit initially, do you have a moment to comment?